agent/scheduler.py

```
@@ -85,15 +105,57 @@ class AgentScheduler:
+        """Reload persisted schedules on boot and re-register their cron triggers.
+
+        Returns the number of jobs rehydrated. Does NOT fire jobs or re-persist
+        them. Safe to call once at startup after construction. Fixes #505 — the
+        12 company cadences + tech-debt burndown now survive a redeploy.
+        """
+        if self._persistence is None:
+            return 0
+        count = 0
+        try:
+            docs = self._persistence.load_all()
+        except Exception as exc:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)